src/main/java/org/sagebionetworks/web/client/widget/table/v2/results/TableQueryResultWidget.java

```diff
@@ -272,8 +273,10 @@ public void onCancel() {
 	 */
 	private void setQueryResults(final QueryResultBundle bundle) {
 		if (cachedFullQueryResultBundle != null) {
+			// Add the cached parts that we didn't request to the new result bundle.
+			// See CACHED_PARTS_MASK for which parts are cached
+			bundle.setQueryCount(cachedFullQueryResultBundle.getQueryCount());
 			bundle.setColumnModels(cachedFullQueryResultBundle.getColumnModels());
-			bundle.setFacets(cachedFullQueryResultBundle.getFacets());
```

Comment:
We didn't previously retrieve the Facets in the query bundle (it wasn't present in the above parts mask)